docs/usage.md

```diff
@@ -0,0 +1,151 @@
+# About
+
+The Data Commons import tool is used to analyze and debug files that are developed
+in the process of importing new datasets to the Data Commons Knowledge Graph.
+
+The tool:
+- Operates on two types of files: instance MCF (.mcf); Template MCF (.tmcf) and corresponding CSV files
+- Performs resolution and, syntax and statistics validations
+- Generates instance MCF from template MCF and corresponding CSV files
+- Generates reports on error/warning counters, stats validation and sample time-series charts
+
+The tool is actively used for all data imports that are included in the Data Commons Knowledge Graph.
+It is under active development, including feature additions and bug fixes.
+
+The tool is a command line application built with Java. See below for usage instructions.
+
+# Usage
+
+Use the import tool from the command line, like so:
+
+  ```bash
+  java -jar <path-to-jar> <mode> <list of mcf/tmcf/csv files>
+  ```
+
+Hint: it can be useful to create an alias for the jar file, such as:
+  ```bash
+  alias dc-import='java -jar <path-to-jar>'
+  ```
+
+This is the form that will be used in the rest of the documentation.
+
+Hint: to access a concise explanation of usage modes and flags, run
+`dc-import --help`
+
+## Usage Modes
+
+In `lint` mode, the import tool validates the artifacts produced for addition to Data Commons. These artifacts include [instance MCF files](https://github.com/datacommonsorg/data/blob/master/docs/mcf_format.md#instance-mcf) and pairs of [template MCF (TMCF)](https://github.com/datacommonsorg/data/blob/master/docs/mcf_format.md#template-mcf) and corresponding CSV files.
+
+In `genmcf` mode, the import tool produces instance MCF files from a pair of TMCF file, and its associated CSV files. This mode performs all validations that the  `lint` mode would have performed.
+
+Both modes generate two output files:
+- `report.json` is a machine-readable output of error/warning counters and details of those logs.
+- `summary_report.html` is a version of `report.json` optimized for viewing in a web browser.
```

Comment:
Not exactly.  It only includes summary counters from `report.json` (not the logs), followed by statistical summaries for sample place (not in `report.json`)